### ISSUE TITLE
Inbox sometimes gets stuck loading infinitely when "unread only" is on

### ISSUE THREAD

**Issue by Sjmarf** [mlemgroup/mlem#1544]
This is still an issue in some cases

**Comment by Sjmarf** [comment]
Repro:
- Start with the "unread only" switched off
- Mark only one item as unread
- Toggle "unread only" on
- Observe that loading symbol gets stuck

**Comment by Sjmarf** [comment]
I'm able to repro this again 🤔

**Comment by EricBAndrews** [comment]
Hmm, I was able to reproduce once but can't do it again. It was the same issue (loader got stuck in `.idle`), but I'm not sure how it got there 🤔 

I'll put the "load more" button back in as a fallback for this--I don't love it as a band-aid but I'm stumped on this one. Maybe it'll go away with [these planned changes](https://github.com/mlemgroup/MlemMiddleware/issues/76)

**Comment by Sjmarf** [comment]
I can consistently repro this using the steps in my comment above on both of my accounts 🤔

**Comment by EricBAndrews** [comment]
Yeah, I had it consistently until I stepped through in the debugger and now I can't get a single repro

**Comment by EricBAndrews** [comment]
Can you try to repro on `eric/idle-debugging`? I've added a bunch of test prints that may shed some light.

**Comment by EricBAndrews** [comment]
Band-aid fix is in place so I'm taking this off 2.0, but keeping it open because the root cause is still unknown
